packages/components/src/components/explore-main-product-cards.stories.ts

```diff
@@ -0,0 +1,19 @@
+import { Meta, StoryObj } from '@storybook/react';
```

Comment:
this story looks great, just saw some outline at bottom 
<img width="474" alt="image" src="https://github.com/user-attachments/assets/43dd76fd-9503-4ec2-a20d-44eb4a0502ba">

Reply:
<img width="911" alt="image" src="https://github.com/user-attachments/assets/c06649ee-45a0-4dc0-b616-080690e4e911">

fixed